.github/workflows/js_tests.yml

```diff
@@ -0,0 +1,32 @@
+name: JavaScript
+on:
+  pull_request:
+    paths:
+      - 'webpack/**'
+      - 'package.json'
+      - '.github/workflows/js_tests.yml'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [10, 12]
```

Comment:
I always like the multi line notation more (as was done in Travis). When you add or remove a version, the diff is smaller and easier to read.